android/app/src/androidTest/java/com/trafi/example/TestPreviews.kt

```diff
@@ -1,11 +1,14 @@
 package com.trafi.example
 
 import androidx.compose.runtime.Composable
-import com.trafi.ui.ButtonPreview
-import com.trafi.ui.ButtonStyledCornerRadiusPreview
+import com.trafi.ui.*
 
 @Suppress("RemoveExplicitTypeArguments")
 val testPreviews = listOf<Pair<@Composable () -> Unit, String>>(
         Pair({ ButtonPreview() }, "ButtonPreview"),
-        Pair({ ButtonStyledCornerRadiusPreview() }, "ButtonStyledCornerRadiusPreview")
+        Pair({ ButtonStyledCornerRadiusPreview() }, "ButtonStyledCornerRadiusPreview"),
+        Pair({ InfoButtonLongTextPreview() }, "InfoButtonLongTextPreview"),
```

Comment:
I wonder if there is value in using the function name as the test name 🤔 See iOS using titles like "Long title" and "Themed" in `InfoButton.swift`. Ideally this naming would even match so we could compare apples to apples across Android & iOS 🌟

Reply:
I updated them to be more similar to iOS, but cannot make the same, because in that case we would have same name "Themed" for all the components and in the final test report there would be no way to distinguish them. Now we have one report for all tests, but we are planning to split it by components in the future, then names could be updated too. This is not the final setup yet :)